<?xml version="1.0"?>
<launch>

 <!-- small description about your node -->
 
 <node pkg="surge_controller" type="surge_controller_node" name="SurgeControllerNode" respawn="false" output="screen">
	<rosparam command="load" file="$(find surge_controller)/config/config_surge_controller.yaml"/>
</node>

</launch>
